src/tests/entities/team.test.ts

```diff
@@ -1,63 +1,147 @@
 import 'jest';
-import { UpdateResult } from 'typeorm';
+import { UpdateResult, SelectQueryBuilder } from 'typeorm';
 import { Team } from '../../entities/team';
 
 /* eslint-disable no-await-in-loop */
+const mockCreateQueryBuilder = {
+  update: jest.fn().mockReturnThis(),
+  set: jest.fn().mockReturnThis(),
+  where: jest.fn().mockReturnThis(),
+  whereInIds: jest.fn().mockReturnThis(),
+  andWhere: jest.fn().mockReturnThis(),
+  execute: jest.fn().mockReturnThis(),
+  select: jest.fn().mockReturnThis(),
+};
+const createQuerySpy = jest
+  .spyOn(Team, 'createQueryBuilder')
+  .mockReturnValue((mockCreateQueryBuilder as Partial<SelectQueryBuilder<Team>>) as SelectQueryBuilder<Team>);
 
-xdescribe('judging', () => {
-  let team: Team;
+const result: UpdateResult = {
+  affected: 0,
+  raw: undefined,
+  generatedMaps: undefined,
+};
 
-  it('decrement reduces activeJudgeCount by 1', async () => {
-    team.activeJudgeCount = 2;
-    await team.save();
-    await team.decrementActiveJudgeCount();
-    let updatedTeam = await Team.findOneOrFail(team.id);
-    expect(updatedTeam.activeJudgeCount).toEqual(1);
+const mockMethod = jest.fn(
+  (): Promise<UpdateResult> => {
+    return Promise.resolve(result);
+  },
+);
 
-    await team.decrementActiveJudgeCount();
-    updatedTeam = await Team.findOneOrFail(team.id);
-    expect(updatedTeam.activeJudgeCount).toEqual(0);
+const mockQueryBuilder = {
+  where: jest.fn().mockReturnThis(),
+  clone: jest.fn().mockReturnThis(),
+  andWhere: jest.fn().mockReturnThis(),
+  orderBy: jest.fn().mockReturnThis(),
+  addOrderBy: jest.fn().mockReturnThis(),
+  getOne: jest.fn().mockReturnThis(),
+};
+
+describe('', () => {
```

Comment:
We need to pass a string in here to describe what we're testing. Looks like the previous one was `judging`... should we add that back here?